CHANGELOG

```diff
@@ -1,6 +1,15 @@
 CHANGELOG
 =========
 
+Version 1.6.0
+=============
+
+* Drop support for Rails 6.x
```

Comment:
Doesn't this necessitate a major?

Reply:
In theory... Yes. Thing is that no users of this gem, out applications, were using Rails 6.0 so I decided to skip the semver bump. I could be persuaded otherwise.